.rubocop.yml

```diff
@@ -29,21 +23,9 @@ Layout/IndentHash:
 Layout/SpaceInsideHashLiteralBraces:
   Enabled: false
 
-Metrics/ClassLength:
-  Enabled: false
-
-Metrics/LineLength:
-  Enabled: false
-
-Metrics/MethodLength:
-  Enabled: false
```

Comment:
OK, if we're going to enable https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Metrics/MethodLength, then let's specify what method length we're shooting for.  How about 
```
Metrics/MethodLength:
    Max: 40
```
and we can scale down from there if we'd like.

Reply:
Sounds good for now.